R/makeApiCall.R

```diff
@@ -139,7 +140,8 @@
 makeApiCall <- function(rcon, 
                         body   = list(), 
                         config = list(),
-                        url    = NULL)
+                        url    = NULL,
+                        success_status_codes = 200L)
```

Comment:
Does "success_status_codes" need `assert_integerish`? I'm not sure it's necessary, but it seems most arguments are checked?

Reply:
Good catch.